com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardFieldView.cs

```diff
@@ -66,7 +67,8 @@ private void DirtyNodes(ModificationScope modificationScope = ModificationScope.
         }
 
         // When the properties are changed, this delegate is used to trigger an update in the view that represents those properties
-        private Action m_inspectorUpdateTrigger;
+        public Action InspectorUpdateTrigger;
```

Comment:
These delegates should be private or else any external class can _set_ them. You should make another public function in this class that calls them so external classes can call them.